go/felix/intdataplane/endpoint_mgr.go

```diff
@@ -62,6 +62,13 @@ type endpointManager struct {
 	// Pending updates, cleared in CompleteDeferredWork.
 	pendingEndpointUpdates map[proto.WorkloadEndpointID]*proto.WorkloadEndpoint
 	pendingIfaceUpdates    map[string]ifacemonitor.State
+
+	// Host endpoint processing.
+	ifaceAddrs               map[string][]string
+	rawHostEndpoints         map[proto.HostEndpointID]*proto.HostEndpoint
```

Comment:
I think I am being consistent already, in that:

- rawHostEndpoints is the raw material, and contains the non-pointer ID
- other places use a pointer, on the grounds that it should be (a bit) more efficient, and that the non-pointer ID that they point to must always exist.